src/Controls/Maps/src/AppHostBuilderExtensions.cs

```diff
@@ -63,7 +63,12 @@ public static IMauiHandlersCollection AddMauiMaps(this IMauiHandlersCollection h
 			handlersCollection.AddHandler<Pin, MapPinHandler>();
 			handlersCollection.AddHandler<MapElement, MapElementHandler>();
 #endif
+
+#if WINDOWS
+			throw new NotImplementedException(".NET MAUI Maps is currently not implemented for Windows. For more information, please see: https://aka.ms/maui-maps-no-windows");
```

Comment:
Maybe instead of an exception that forces the user to condition out, we can use the `[SupportedOS]` attributes? 

Rather, what is the expected code now that this is here to use the community toolkit?

```cs
builder
    .UseMauiApp<App>()
#if !WINDOWS
    .AddMauiApps()
#endif
    .UseMauiToolkit()
```

Reply:
We don't have the Toolkit installed by default, so people would have to add that, then follow this: https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/get-started?tabs=CommunityToolkitMauiMaps#using-the-nuget-packages

Basically adding

```csharp
var builder = MauiApp.CreateBuilder();
builder
    .UseMauiApp<App>()
    .UseMauiCommunityToolkitMaps("YOUR_API_KEY")
```

That's just a bit too much to add here?

Where would be put the SupportedOS on? The extension method then for this? I guess that could work, although it's still easier to overlook and then you'll still run into a much more confusing exception? @mattleibow